File: DigitalPet/app/src/main/java/thedigialex/digitalpet/model/entities/UserTrainingEquipment.kt
package thedigialex.digitalpet.model.entities
import com.google.gson.annotations.SerializedName

data class UserTrainingEquipment(
    val id: Int,
    @SerializedName("user_id") val userId: Int,
    @SerializedName("training_equipment_id") val trainingEquipmentId: Int,
    @SerializedName("stat_increase") val statIncrease: Int,
    val level: Int,
    @SerializedName("training_equipment") val trainingEquipment: TrainingEquipment
)

File: digital-monster/app/Models/UserTrainingEquipment.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class UserTrainingEquipment extends Model
{
    protected $fillable = [
        'user_id',
        'training_equipment_id',
        'level',
    ];

    public function user()
    {
        return $this->belongsTo(User::class);
    }

    public function trainingEquipment()
    {
        return $this->belongsTo(TrainingEquipment::class);
    }
}
